joomla/joomla-cms
# New Module select

#### Summary of Changes

Remove the extra space after the name of the module in the link
#### Testing Instructions

Go to create a new module and check that the extra space has been removed at the end of the link

PS I have no idea why the space is appearing just because the closing a tag is on a new line but removing the new line fixes it
